packages/webapp/src/components/Form/Unit/index.jsx

```diff
@@ -58,7 +58,7 @@ const Unit = ({
   onBlur,
   hasLeaf,
   autoConversion,
-  onChange: additionalOnChange,
+  onChange: onInputChange,
```

Comment:
Sorry 😭 I meant renaming `onChange` itself to differentiate it from `onChangeUnitOption`.

Reply:
Oh haha 😂 It won't have the same convention as the other inputs in that case though; that's okay?

Reply:
I guess how I think about `<Unit />` was different - it wasn't a type of input for me 😂 I'm okay as is!

Reply:
I did change it; thank you!